style.css

```diff
@@ -53,7 +57,13 @@ body{
     color: var(--primary-maroon);
 }
 
+.background-light-grey{
+    background-color: var(--light-grey);
+}
 
+.position-static{
+    position: static !important;
```

Comment:
avoid using !important, rather play with specificity.

Reply:
Thanks for the review, will surely do the changes :)